src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private static final String DEFAULT_DRAW_MESSAGE =
+            "Figure: %s, area: %s sq.units, top: %s units, "
+                    + "bottom: %s units, height: %s units, color: %s";
+    private int topLength;
+    private int bottomLength;
+    private int height;
+
+    public IsoscelesTrapezoid(int topLength, int bottomLength, int height, Color color) {
+        setFigure(FigureType.ISOSCELES_TRAPEZOID);
+        setTopLength(topLength);
+        setBottomLength(bottomLength);
+        setHeight(height);
+        setColor(color);
+    }
+
+    public IsoscelesTrapezoid() {
+        setFigure(FigureType.ISOSCELES_TRAPEZOID);
+        setTopLength(2);
+        setBottomLength(2);
+        setHeight(2);
+        setColor(Color.BLUE);
+    }
+
+    @Override
+    public double getCalculatedArea() {
+        return ((bottomLength * topLength) * height) / 2.0;
```

Comment:
The formula for calculating the area of an isosceles trapezoid is incorrect. It should be `((topLength + bottomLength) * height) / 2.0` instead of `((bottomLength * topLength) * height) / 2.0`. Please correct this to ensure the area is calculated accurately.